docs-kits/kits/Connector Kit/Adoption View/working-with-policies.md

```diff
@@ -0,0 +1,126 @@
+Verifiable Presentation (VP). A VP is a set of Verifiable Credentials (VCs) that a Consumer may have been issued by a
+trusted third party. In this VC, a Provider may find additional information if a Consumer is eligible to pass a certain
+`odrl:Constraint`. Transfer of VPs is handled automatically between the Participants' Identity Wallets by means of the
+[IATP VPP](https://github.com/eclipse-tractusx/identity-trust/blob/main/specifications/verifiable.presentation.protocol.md).
+
+If, for example, the Consumer tries to negotiate for an Offer that is extended only to interested
+parties from civil society (like an NGO), simply pretending to be an NGO shouldn't be enough. It has to be verified and
+can be verified with help of the VP. For that, a "NonGovernmentalOrganizationCredential" could be issued during the
+onboarding process according to the role a Participant assumes in the Dataspace. For more on the interrelation between
+Credentials and Constraints, refer to standards CX-0018 and CX-0149.
+
+A offer containing a `odrl:Use` permission with a `odrl:Constraint` could (in a hypothetical scenario) look like this:
+
+```json
+{
+  "@context": {
+    "odrl": "http://www.w3.org/ns/odrl/2/"
+  },
+  "@id": "Y29udHJhY3QtYmlsYXRlcmFsLXBhcGVyLWV4YW1wbGUtNA==:anNvbi0xLXBhcGVy:Y2JhMGFmM2QtZTIwYi00ZjExLThlZmQtM2FhMzcyZmI1Nzhh",
+  "@type": "odrl:Set",
+  "odrl:permission": {
+    "odrl:action": {
+      "odrl:type": {
+        "@id": "http://www.w3.org/ns/odrl/2/use"
+      }
+    },
+    "odrl:constraint": {
+      "odrl:and": [
+        {
+          "odrl:leftOperand": "https://w3id.org/some-dataspace/role",
+          "odrl:operator": {
+            "@id": "odrl:eq"
+          },
+          "odrl:rightOperand": "NGO"
+        }
+      ]
+    }
+  },
+  "odrl:prohibition": [],
+  "odrl:obligation": []
+}
+```
+
+The Constraint would in that case be checked against the `"type"` property, attesting the information that the presented
+VC does in fact attest that the holder is a Non-Governmental Organization. A Provider Connector would have to have
+appropriate logic implemented to extract relevant information from the VP to satisfy the `odrl:Constraint`.
+
+```json
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://w3id.org/mydataspace/credentials/v1.0.0"
+  ],
+  "id": "1f36af58-0fc0-4b24-9b1c-e37d59668089",
+  "type": [
+    "VerifiableCredential",
+    "NonGovernmentalOrganizationCredential"
+  ],
+  "issuer": "did:web:com.example.issuer",
+  "issuanceDate": "2021-06-16T18:56:59Z",
+  "expirationDate": "2022-06-16T18:56:59Z",
+  "credentialSubject": {
+    "id": "did:web:com.example.participant",
+    "holderIdentifier": "BPNL000000001"
+  }
+}
+```
+
+## Notice
+
+This work is licensed under the [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).
+
+- SPDX-License-Identifier: CC-BY-4.0
+- SPDX-FileCopyrightText: 2023 Contributors of the Eclipse Foundation
```

Comment:
Please unify across Adoption, Operation, Dev. View: either "Contributors of the Eclipse Foundation" or listing out individual company names - not both. (see also paragraphs below).